cfg/18.09/definitions.yaml

```diff
@@ -0,0 +1,1824 @@
+
+  - id: 5.17
+    description: "Ensure that host devices are not directly exposed to containers (Not
+    Scored)"
+    type: manual
+    remediation: |
+      You should not directly expose host devices to containers. If you do need to expose host
+      devices to containers, you should use granular permissions as appropriate to your
+      devices to containers, you should use granular permissions as appropriate to your
+      For example, do not start a container using the command below:
+      docker run --interactive --tty --device=/dev/tty0:/dev/tty0:rwm --
+      device=/dev/temp_sda:/dev/temp_sda:rwm centos bash
+      You should only share the host device using appropriate permissions:
+      docker run --interactive --tty --device=/dev/tty0:/dev/tty0:rw --
+      device=/dev/temp_sda:/dev/temp_sda:r centos bash
+    scored: false
+
+  - id: 5.18
+    description: "Ensure that the default ulimit is overwritten at runtime, only if needed
+    (Not Scored)"
+    type: manual
+    remediation: |
+      You should only override the default ulimit settings if needed in a specific case.
+      For example, to override default ulimit settings start a container as below:
+      docker run --ulimit nofile=1024:1024 --interactive --tty centos /bin/bash
+    scored: true
+
+  - id: 5.19
+    description: "Ensure mount propagation mode is not set to shared (Scored)"
+    type: manual
+    remediation: |
+      Do not mount volumes in shared mode propagation.
+      For example, do not start container as below:
+      docker run <Run arguments> --volume=/hostPath:/containerPath:shared
+      <Container Image Name or ID> <Command>
+    scored: true
+
+  - id: 5.20
+    description: "Ensure that the host's UTS namespace is not shared (Scored)"
+    audit: docker ps --quiet --all | xargs docker inspect --format '{{ .Id }}:UTSMode={{ .HostConfig.UTSMode }}'
+    use_multiple_values: true
+    tests:
+      test_items:
+      - flag: "UTSMode=host"
+        set: false
+    remediation: |
+      You should not start a container with the --uts=host argument.
+      For example, do not start a container as below:
+      docker run --rm --interactive --tty --uts=host rhel7.2
+    scored: true
+
+  - id: 5.21
+    description: "Ensure the default seccomp profile is not Disabled (Scored)"
+    audit: docker ps --quiet --all | xargs docker inspect --format '{{ .Id }}:SecurityOpt={{ .HostConfig.SecurityOpt }}'
+    use_multiple_values: true
+    tests:
+      bin_op: and
+      test_items:
+      - flag: "SecurityOpt"
+        compare:
+          op: nothave
+          value: "seccomp:unconfined"
+        set: true
+      - flag: "SecurityOpt"
+        compare:
+          op: nothave
+          value: "seccomp=unconfined"
+        set: true
+    remediation: |
+      By default, seccomp profiles are enabled. You do not need to do anything unless you want
+      to modify and use the modified seccomp profile.
+    scored: true
+
+    # Revisit
```

Comment:
There are a few more in this section marked "Revisit"

Reply:
Just checked them all, they all work or manual, I guess that the previous person worked on it want the check option for doing it automatically.
For this PR its seems fine to me, and I should look at it again in a new pr because its all changes that need to be done in older versions as well.
WDYT @lizrice ?